velox/functions/prestosql/tests/StringFunctionsTest.cpp

```diff
@@ -223,9 +215,16 @@ class StringFunctionsTest : public FunctionBaseTest {
       output += ")";
       return output;
     };
+
+    // Evaluate 'concat' expression and verify no excessive memory allocation.
+    auto numAllocsBefore = pool()->getMemoryUsageTracker()->getNumAllocs();
```

Comment:
Could you comment what are the two memory allocations? Ensure write and aggregated string copy? Thanks!

Reply:
Allocation of values buffer (StringViews) and strings buffer.